python/hopsworks_common/connection.py

```diff
@@ -359,8 +432,13 @@ def close(self) -> None:
             conn.close()
             ```
         """
+        from hsfs import engine
+
         OpenSearchClientSingleton().close()
         client.stop()
+        engine.stop()
+        self._feature_store_api = None
+        self._model_api = None
```

Comment:
model_serving_api, model_registry_api?

Reply:
It was this way in hsml before the merge: https://github.com/logicalclocks/machine-learning-api/blob/main/python/hsml/connection.py#L200

Also, I would prefer to remove `_model_api = None` instead of adding them.